uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Form/AbsenceType.php

```diff
@@ -140,6 +143,15 @@ private function addBasicDetailsFields() {
         ['disabled' => 'disabled']
       );
     }
+    $this->addSelect(
+      'calculation_unit',
+      [
+        'options' => $this->getCalculationUnitOptions(),
+        'option_url' => NULL,
```

Comment:
Is it really necessary to set the option_url to null?

Reply:
Well, If it's not set to Null, it adds a link to edit/create option values for the calculation_unit  in front of the Calculate Leave in field.

Reply:
Why don't we have the same problem with other select fields in this form, like this one: https://github.com/civicrm/civihr/blob/la-milestone-3/uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Form/AbsenceType.php#L216-L219

Reply:
Thats because `accrual_expiration_unit` does not reference an option group.